SonarQube.TeamBuild.PostProcessor/Program.cs

```diff
@@ -116,7 +116,7 @@ private static bool CheckEnvironmentConsistency(AnalysisConfig config, TeamBuild
         private static ProjectInfoAnalysisResult InvokeSonarRunner(AnalysisConfig config, ILogger logger)
         {
             ISonarRunner runner = new SonarRunnerWrapper();
-            ProjectInfoAnalysisResult result = runner.Execute(config, logger);
+            ProjectInfoAnalysisResult result = runner.Execute(config, Enumerable.Empty<string>() /* todo */, logger);
```

Comment:
I'll changed the post-processor next to validate and pass through command line args.